scripts/docker-build.sh

```diff
@@ -2,6 +2,7 @@
 
 set -euo pipefail
 
-DOCKER_BUILDKIT=1 docker buildx build --platform=linux/amd64 -f docker/Dockerfile \
+DOCKER_BUILDKIT=1 docker buildx build --platform=linux/amd64 -f Dockerfile \
+  --build-arg PIP_VERSION="$PIP_VERSION" \
   --progress plain \
-  -t ml-inference-dev:latest .
+  -t unstructured-inference-dev:latest .
```

Comment:
I left it ml-inference as far as the app/image name was concerned when I renamed the repo. Changing the name is fine, but references would need to be aligned in `deployment-tools` and `devops` (and maybe others that aren't coming to mind right now).

Reply:
discussed async. `unstructured-inference-dev` is at least consistent with other repos and should not lead to image-name confusion.